content/en/methods/instance.md

```diff
@@ -28,7 +28,8 @@ Obtain general information about the server.
 **Returns:** [Instance]({{< relref "entities/Instance" >}})\
 **OAuth:** Public\
 **Version history:**\
-4.0.0 - added
+4.0.0 - added\
+4.3.0 - added `configuration.vapid.public_key`
```

Comment:
should probably be added to the example response below

Reply:
It is there: https://github.com/ThisIsMissEm/mastodon-documentation/blob/chore/document-oauth-changes/content/en/methods/instance.md?plain=1#L63-L65